source/includes/code-examples/update-many/Restaurant.cs

```diff
@@ -0,0 +1,24 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+namespace UsageExamples.UpdateMany;
+
+// start-model
+public class Restaurant
```

Comment:
Question: Can you simplify the restaurant model to only include fields used in the example? Perhaps only the "cuisine" and "_id" value? If possible, I think this will make the page much shorter without loosing explanatory power.

Reply:
Good idea.